scripts/create_compat.zs

```diff
@@ -1,6 +1,15 @@
 // Create integration provided by MobiusFlip
-
 #snip modnotloaded create
+
+#snip modloaded createtweaker
+import crafttweaker.api.events.CTEventManager;
+CTEventManager.register<crafttweaker.api.event.entity.player.MCPlayerLoggedInEvent>((event) => {
+     var player = event.player;
+     player.sendMessage("You need create tweaker dummy: https://www.curseforge.com/minecraft/mc-mods/createtweaker");

```

Comment:
Less aggressive please lmao

Reply:
While hilarious, the message doesn't actually explain *why* you need CreateTweaker. Sure, you can derive it from context, but maybe someone added create previously and forgot, or a friend handed them the pack. More information is better.

Reply:
Pfft yeah fair. Changing it now

Reply:
Ok, done